### PR TITLE
[ENG-876] Feature completeness

### PR DESCRIPTION
**Pull request by tabergma**
## Description
- added flows and e2e tests to ensure feature completeness (see [notion page](https://www.notion.so/rasa/rasa-calm-demo-feature-completeness-c5d76a786fa949b18146450b3b0e149a))

## TODOs

- [x] compared flaky tests with the [known list of flaky tests steps](https://www.notion.so/rasa/Flaky-E2E-Test-Steps-63864d3d8c7b4427a0f3df8052e39f21)

### CODE REVIEW

**Comment by radovanZRasa** [comment]
This is awesome 💯 !!!

File: domain/flows/authenticate_user.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    type: text
+    mappings:
+      - type: custom
+
+responses:
+  utter_authentication_failed:
+    - text: "Authentication failed. Please try again."
+  utter_authentication_successful:
+    - text: "You have successfully logged in."
+  utter_ask_user_name:
```
Comment (Northwind01) [inline domain/flows/authenticate_user.yml:20]: We should also try to make the "" included or not consistently for `- text: "..."`

File: domain/flows/check_portfolio.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+          title: stocks
+        - payload: bonds
+          title: bonds
+        - payload: mutual_funds
+          title: mutual funds
+  utter_portfolio_options_found:
+    - text: "Your {portfolio_type} portfolio: {portfolio_options}"
+  utter_portfolio_options_not_found:
+    - text: You don't have any {portfolio_type} investments.
+
```
Comment (Northwind01) [inline domain/flows/check_portfolio.yml:38]: e.g. here we have it with and without ""

Comment (tabergma) [reply domain/flows/check_portfolio.yml:38]: " is needed because of the : in the text

**Review by varunshankar — APPROVED** [review]

**Pull request merged**